lib/validations.js

```diff
@@ -66,6 +66,10 @@
 
   validations.required = function(input) {
     return validations.validate(input, function(text) {
+      if(typeof text !== 'string') {
```

Comment:
Hmm, good point. Tests are even considering that either. Thinking about it now, it should just check if the value is not null, undefined, or empty.